common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -781,6 +781,24 @@ colored_emblem_texture_lists = {
 			}
 			100 	= "ce_kirintor_eye.dds"
 		}
+		
+		### Death
+		special_selection = {
+			trigger = {
+				death_coa_trigger = yes
+			}
+			100 	= "ce_abomination.dds"
+			100 	= "ce_dk_helmet.dds"
+			100 	= "ce_necropolis.dds"
+			100 	= "ce_ram_skull.dds"
+			100 	= "ce_scream.dds"
+			100 	= "ce_skull1.dds"
+			100 	= "ce_skull1_fire.dds"
+			100 	= "ce_skull2.dds"
+			100 	= "ce_skull3.dds"
+			100 	= "ce_skull3_jaw.dds"
+			100 	= "ce_spider.dds"
```

Comment:
Should be lowered.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126882318-2bce9e34-f01c-4c89-94bd-b2358a25f51f.png)

</details>